auth-settings.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+
+  // authTypeInbound can be 'OAuth2' or 'Signature'
+  authTypeInbound: process.env.AUTH_TYPE_INBOUND,
+  //OAuth config for inbound
+  oauthConfig: {
+    tokenUrl: process.env.CONVO_INSTANCE_URL
+        || `${process.env.CONVO_WEBHOOK_URL && new URL(process.env.CONVO_WEBHOOK_URL).origin}/oauth/token`,
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET,
+  },
+  //for authTypeInbound Signature
+  secret: process.env.SHARED_SECRET,
+
+  // authType can be 'OAuth2' or 'API-Token'
+  authTypeOutbound: process.env.AUTH_TYPE_OUTBOUND,
+
+  // for outbound request API-Token verification (i.e. requests coming from MC)
+  accessToken: process.env.ACCESS_TOKEN,
+
+  // This is the OAuth client configuration (adapter acting as client to MC as OAuth server)
```

Comment:
Add a `.` to the end here.  Don't abbreviate MC.  Update the parenthetical to be more fluid and clear:
```
This is the OAuth 2.0 configuration used by Medallia Conversations to connect with the channel adapter.
```